.travis.yml

```diff
@@ -23,6 +23,7 @@ script:
   - ./httpstat -X DELETE http://httpbin.org/delete
   - ./httpstat https://www.google.com/
   - ./httpstat https://www.google.com:443/
+  - ./httpstat -H "Host: www.google.com" https://$(dig +short www.google.com):443/
```

Comment:
Can we make this a bit more resistant to failure? If Google ever adds multiple A records this test will fail. How do you feel about `dig +short www.google.com | head -n 1`?